src/main/java/org/openstreetmap/atlas/utilities/filters/AtlasEntityPolygonsFilter.java

```diff
@@ -0,0 +1,520 @@
+
+    public static AtlasEntityPolygonsFilter forConfiguration(final Configuration configuration)
+    {
+        return forConfiguration(configuration, DEFAULT_INTERSECTION_DECIDING);
+    }
+
+    public static AtlasEntityPolygonsFilter forConfiguration(final Configuration configuration,
+            final IntersectionDeciding intersectionDeciding)
+    {
+        return forConfigurationValues(configuration.get(INCLUDED_POLYGONS_KEY).value(),
+                configuration.get(INCLUDED_MULTIPOLYGONS_KEY).value(),
+                configuration.get(EXCLUDED_POLYGONS_KEY).value(),
+                configuration.get(EXCLUDED_MULTIPOLYGONS_KEY).value(), intersectionDeciding);
+    }
+
+    public static AtlasEntityPolygonsFilter forConfigurationValues(
+            final Map<String, List<String>> includePolygonMap,
+            final Map<String, List<String>> includeMultiPolygonMap,
+            final Map<String, List<String>> excludePolygonMap,
+            final Map<String, List<String>> excludeMultiPolygonMap,
+            final IntersectionDeciding intersectionDeciding)
+    {
+        final List<Polygon> includePolygons;
+        final List<Polygon> excludePolygons;
+        final List<MultiPolygon> includeMultiPolygons;
+        final List<MultiPolygon> excludeMultiPolygons;
+
+        includePolygons = includePolygonMap != null ? getPolygonsFromFormatMap(includePolygonMap)
+                : Collections.EMPTY_LIST;
+        includeMultiPolygons = includeMultiPolygonMap != null
+                ? getMultiPolygonsFromFormatMap(includeMultiPolygonMap) : Collections.EMPTY_LIST;
+        excludePolygons = excludePolygonMap != null ? getPolygonsFromFormatMap(excludePolygonMap)
+                : Collections.EMPTY_LIST;
+        excludeMultiPolygons = excludeMultiPolygonMap != null
+                ? getMultiPolygonsFromFormatMap(excludeMultiPolygonMap) : Collections.EMPTY_LIST;
+
+        if (!includePolygons.isEmpty() || !includeMultiPolygons.isEmpty())
+        {
+            return Type.INCLUDE.polygonsAndMultiPolygons(intersectionDeciding, includePolygons,
+                    includeMultiPolygons);
+        }
+        else
+        {
+            return Type.EXCLUDE.polygonsAndMultiPolygons(intersectionDeciding, excludePolygons,
+                    excludeMultiPolygons);
+        }
+    }
+
+    public static AtlasEntityPolygonsFilter forConfigurationValues(
+            final Map<String, List<String>> includePolygonMap,
+            final Map<String, List<String>> includeMultiPolygonMap,
+            final Map<String, List<String>> excludePolygonMap,
+            final Map<String, List<String>> excludeMultiPolygonMap)
+    {
+        return forConfigurationValues(includePolygonMap, includeMultiPolygonMap, excludePolygonMap,
+                excludeMultiPolygonMap, DEFAULT_INTERSECTION_DECIDING);
+    }
+
+    private static StringConverter<Optional<List<MultiPolygon>>> getMultiPolygonConverterForFormat(
```

Comment:
There seems to be quite a bit of overlap from getMultiPolygon* and getPolygon* below here.  Would an interface or base class reduce code duplication?

Reply:
I wanted to do something of that nature, but because Polygon and Multipolygon  don't share a base class, nor are their converts exactly the same, this was the best solution I could come up with.

Reply:
We could have a generic base class called ```PolygonStringFormatConverter``` that exposes a ```handleAtlas```, ```handleGeoJson```(and so forth) and have common logic in the base class and implementations will implement the different handlers.